### ISSUE TITLE
On print conditionally show instructions.

### ISSUE THREAD

**Issue by JohnLukeBentley** [robinst/ausballot#2]
Reply to https://www.reddit.com/r/AustralianPolitics/comments/umwy7g/ausballot_preview_your_ballot_papers_and_prepare/i8eyvhj/

> Hmm I'd like to keep the instructions there. There's nothing preventing anyone from printing the page with the ballots empty, so in that case I think it would make sense to have the instructions still there.

A good consideration. (Arguably it is unnecessarily complex but) you could make the instructions conditional:

```
If (IsBallotsValidlyFilled()) {
  HideInstructions() 
} else {
  ShowInstructions()
}
```

**Comment by robinst** [comment]
Yeah that would work.. I don't think we're saving that much space by removing that line, but maybe if the ballots are even bigger next year we should consider shrinking the print size a bit more yeah :).

Thanks, I'm going to close this but keep it in mind in the future.

**Comment by JohnLukeBentley** [comment]
It's not about saving space (that may or may not be a tangential benefit). It's about what might be confusing to a user.

As a user you expect your software output to intelligently change based on your inputs. If, as a user, you've validly filled out your ballots and you see instructions on how to fill out your ballot, that would seem to be a signal from the software that your ballot is incomplete.

On the other hand if you haven't filled out your ballot then instructions remain relevant. And, as you mention, there might be the case where the user wants to print out the empty ballot and fill it out manually.

> Thanks, I'm going to close this but keep it in mind in the future.

(Even though you've open sourced it) it's your repo and you have the right to close tickets according to whatever principle, or even whim, as you like. However, if the issue is something to keep in mind that would normally entail that the issue is "open". Might reopening the ticket but labeling it "Next election" (or whatever) better serve your intentions?

**Comment by JohnLukeBentley** [comment]
(As an aside irrelevant to this issue) Robin I can confirm that this all worked perfectly in my electorate on election day. That is, the ausballot printouts matched the real world ballots (intentional omission of the gaggle of individual senate candidates aside). So that's one in-production testing data point. :)

**Comment by robinst** [comment]
Yay, that's nice to hear :)!
